components/search/SearchResult.vue

```diff
@@ -29,11 +29,16 @@
         </template>
```

Comment:
The name of this component is a bit misleading now as it's not always used for search results. Perhaps rename to something like `ListSearchResult.vue`

Reply:
I think that's more confusing with us also having a component SearchResultsList.

Reply:
Fair point. Let's leave it for now and revisit if and when there is a case for reuse of this component, e.g. as a general record metadata + thumbnail preview component.